### DISCUSSION TITLE
Integration: Node.js/Typescript OpenAI SDK

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#844] [Ideas]
Proposal: build drop-in integration for JS OpenAI SDK similar to the one for Python as it makes it super easy to get started with Langfuse by changing a single line of code (the import).

More details on the Python integration: https://langfuse.com/docs/openai

**Comment by arvinxx** [comment]
Really need it from [LobeChat](https://github.com/lobehub/lobe-chat/) 🙋

**Comment by hassiebp** [comment]
@arvinxx We've just released the JS integration for the OpenAI SDK in the [Langfuse JS SDK 3.6.0](https://github.com/langfuse/langfuse-js/releases/tag/v3.6.0) 🎉 Feel free to try it out and let us know your feedback 🙏🏾

[Link to release](https://github.com/langfuse/langfuse-js/releases/tag/v3.6.0)
[Link to docs](https://langfuse.com/docs/integrations/openai/js/get-started)
